### DISCUSSION TITLE
Speak readings minor bug

### DISCUSSION THREAD

**Discussion by osbragina** [NightscoutFoundation/xDrip#2520] [General]
Hi all, installed xDrip+ today and loving it. Noticed a minor so wanted to report. I am using a feature Speak readings and noticed the following:
- If the speak trend arrow name is ticked all good, but if it is unticked the system reads the number only before the dot. So for example it says "nine point" and doesn't say "nine point five".
- If I change the language in "custom locale" field the system translates the number only but the trend arrow name is pronounced in english
Thanks again to the team for a great app!

**Comment by Navid200** [comment]
Would you please clarify which xDrip version you have experienced this with?
The easiest way is to post a screenshot of the classic status page.  https://navid200.github.io/xDrip/docs/xDrip-Version

**Reply by osbragina** [reply]
![Screenshot_20221204-035619_xDrip+](https://user-images.githubusercontent.com/119734167/205452508-70ec7e92-17c5-4eab-bc4f-bf86ce9fcf3b.jpg)

**Reply by Navid200** [reply]
I am using the latest Nightly build (November 27, 2022).
I just enabled speak readings.
Then, I disabled speak trend arrow name.   It still pronounces the fraction.  

Do you mind restarting the phone?
If it still does not pronounce the fraction, would you please update to the latest Nightly?
All you need to do for that is to download this file: https://github.com/NightscoutFoundation/xDrip/releases/download/2022.11.27/xDrip-plus-20221127-7352414.apk
and install it.  Please don't uninstall your existing xDrip.
And it will be as easy to go back if you decide to do so.  Just download the version you like and install over.

I am focused on the first item you have mentioned.  Please let's go one item at a time.

**Reply by osbragina** [reply]
Hi, I am really sorry but I tested it again and it actually works on my phone but doesn't work on my child's phone regardless of the trend arrow tick. I was using both phones yesterday and got confused. We have the same app version on both phones so it must be something with her phone or maybe because I am a follower.
Sorry again.
![IMG_20221204_140124_486](https://user-images.githubusercontent.com/119734167/205472266-f4784fa7-4561-4755-a9f9-700489c88b58.jpg)

**Reply by Navid200** [reply]
That is not a problem at all.

How about the second issue?
Is that also working fine on your phone?

**Reply by osbragina** [reply]
No the second issue is still a thing on both phones. I used ru _Ru language. By the way I found that first issue is only appearing if the language is en_AU.

**Comment by Navid200** [comment]
I am not sure what you mean by if I change the custom language in custom local.

Under normal circumstances, you choose a language for xDrip.  Then, you enable speak readings.  And you choose the same language in speak readings.
Do you experience any problems if you do that?

**Reply by osbragina** [reply]
There is a field called "Custom Locale for Spoken Readings". When I change the language there to russian, it reads the number in russian and trend arrow in English.

![Screenshot_20221212_162811_xDrip+](https://user-images.githubusercontent.com/119734167/206967985-ce79e1c8-3abf-4080-aa9b-8a918910ca62.jpg)


![Screenshot_20221212_162845_xDrip+](https://user-images.githubusercontent.com/119734167/206967995-156ba115-f430-407e-b5c0-6c527f3f6fa7.jpg)

**Reply by Navid200** [reply]
But, Russian is not a custom localization of English.

I asked a question.  You did not answer.
